deploy/operator.yaml

```diff
@@ -41,7 +41,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
-        image: perconalab/percona-server-mysql-operator:main
+        image: perconalab/percona-server-mysql-operator:k8sps-103-add-printer-columns
```

Comment:
please remove all your changes connected with local tests

Reply:
Yes, I was thinking about it. Should I remove them by hand or some script will do it?

Reply:
Yes, I was thinking about it. Should I remove them by hand or some script will do it?

Reply:
You should not add it when you push PR, your code will be tested correctly without these changes.

Reply:
Should I generate CRDs? Is there some specific make command(s) I should run?

Reply:
Fixed this with the help of @tplavcic and @egegunes .